packages/flutter_driver/test_driver/success_test.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import '../test/common.dart';
+
+void main() {
+  // Tooling requires a single trivial test
+  test('it runs a test', () {
+    expect(true, true);
```

Comment:
do we need this line?

Reply:
I want to actually run a test to make sure the `pub run test` logic works

Reply:
oh, you mean the expect? I guess not ... but a totally empty test seemed weird